vxquery-xtest/src/test/resources/cat/LibrariesInJSONiq.xml

```diff
@@ -40,4 +40,9 @@
       <query name="flatten" date="2016-07-20"/>
       <output-file compare="Text">flatten.txt</output-file>
    </test-case>
+    <test-case name="remove-keys" FilePath="Json/Libraries/" Creator="Riyafa Abdul Hameed">
```

Comment:
Please add two more tests: use a string and a empty sequence for the keys value..